x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/pipelines/ml_inference/text_expansion_callout.tsx

```diff
@@ -182,6 +185,90 @@ export const ModelDeploymentInProgress = ({
+}: Pick<TextExpansionCallOutState, 'dismiss' | 'isDismissable' | 'isStartButtonDisabled'>) => {
+  const { startTextExpansionModel } = useActions(TextExpansionCalloutLogic);
+
+  return (
+    <EuiPanel color="success">
+      <EuiFlexGroup direction="column" gutterSize="s">
+        <EuiFlexItem grow>
+          <EuiFlexGroup direction="row" gutterSize="s" alignItems="center">
+            <EuiFlexItem grow={false}>
+              <EuiIcon type="checkInCircleFilled" />
+            </EuiFlexItem>
+            <EuiFlexItem grow>
+              <EuiTitle size="xs">
+                <h4>
+                  <FormattedMessage
+                    id="xpack.enterpriseSearch.content.index.pipelines.textExpansionCallOut.deployedTitle"
+                    defaultMessage="Your ELSER model has deployed but not started." />
+                </h4>
+              </EuiTitle>
+            </EuiFlexItem>
+            {isDismissable && (
+              <EuiFlexItem grow={false}>
+                <TextExpansionDismissButton dismiss={dismiss} />
+              </EuiFlexItem>
+            )}
+          </EuiFlexGroup>
+        </EuiFlexItem>
+        <EuiFlexItem grow>
+          <EuiText>
+            <FormattedMessage
+              id="xpack.enterpriseSearch.content.index.pipelines.textExpansionCallOut.deployedBody"
+              defaultMessage="You may start the model in a single-threaded configuration for testing, or tune the performance for a production environment." />
+          </EuiText>
+        </EuiFlexItem>
+        <EuiFlexItem grow>
+          <EuiFlexGroup direction="row" gutterSize="s" alignItems="center">
+            <EuiFlexItem>
+              <EuiButton
+                aria-label={i18n.translate(
+                  'xpack.enterpriseSearch.content.index.pipelines.textExpansionCallOut.startModelButton',
+                  { defaultMessage: 'Start ELSER model' }
+                )}
+                color="success"
+                disabled={isStartButtonDisabled}
+                iconType="playFilled"
+                onClick={() => startTextExpansionModel(undefined)}
+              >
+                {i18n.translate(
+                  'xpack.enterpriseSearch.content.indices.pipelines.textExpansionCallOut.startModelButton.label',
+                  {
+                    defaultMessage: 'Start single-threaded',
+                  }
+                )}
+              </EuiButton>
+            </EuiFlexItem>
+            <EuiFlexItem>
+              <EuiButtonEmpty
+                aria-label={i18n.translate(
+                  'xpack.enterpriseSearch.content.index.pipelines.textExpansionCallOut.fineTuneModelButton',
+                  { defaultMessage: 'Fine-tune ELSER model' }
+                )}
+                iconSide="left"
+                iconType="wrench"
+                onClick={() => KibanaLogic.values.navigateToUrl('/app/ml/trained_models', {
+                  shouldNotCreateHref: true,
+                })}
```

Comment:
I need some guidance on the proper use of `navigateToUrl()`:
- Currently it opens in the same tab, how can I make it target a new tab?
- Is it OK to hardcode the path, or should I extract it to a constant?

Reply:
@julianrosado Is the UX goal to stay on the same panel here, or open a new tab for the Trained Models page (and fine-tuning the launch of the model)?
Note if we navigate away, the status change won't get reflected here until the user reloads the page.

Reply:
> Currently it opens in the same tab, how can I make it target a new tab?

May have other better solution but I think you could also use  history from `KibanaLogic` values and create an href 
```
const { navigateToUrl, history } = useValues(KibanaLogic);



<EuiLink  href={history.createHref(parsePath('/app/ml/trained_models'))} target="_blank">
View this link
</EuiLink>
```
see  [external links doc](https://elastic.github.io/eui/#/navigation/link#external-links) from Elastic Eui

Reply:
We shouldn't be opening links within Kibana in separate tabs for users, that's a poor accessibility practice as screen reader users lose all context, and it can be a jarring experience for other users who expect links to just keep them in their current tab. If users want to open a link in a new tab, they can do that themselves.